manim/mobject/probability.py

```diff
@@ -177,37 +182,37 @@ def __getitem__(self, index):
         return self.split()[index]
 
 
-class BarChart(VGroup):
-    """This is a class for Bar Charts.
+class BarChart(Axes):
+    """This is a class for easily creating a Bar Chart. The only required parameter is a list of `values`. Everything else is automatically calculated to have a good looking on screen.
 
     Parameters
     ----------
     values
-        The values for the bar chart.
-    height
-        The height of the axes.
-    width
-        The width of the axes.
-    n_ticks
-        Number of ticks.
-    tick_width
-        Width of the ticks.
-    label_y_axis
-        Y axis label
-    y_axis_label_height
-        Height of the label.
-    max_value
-        Maximum value of the data.
+        A list of values for each `bar`. It also accepts negative values.
+    bar_names
+        A list of names for each `bar`. It is optional to match the `values` list length.
+    x_length
+        The x_axis length. If `None` it is automatically adjusted depending on the number of values and the screen frame width.
+    x_label_scale_value
+        The scale value for `x_labels`. If `None` it is automatically calculated depending on the bar's width.
```

Comment:
With #1694, this should be a font_size. Also, since we're inheriting from `Axes`, maybe just have the user control this on their own via `x_axis_config`? It'd help make this class a little more concise + avoid uncessarily defining parameters.